modules/siddhi-extensions/regex/src/test/java/org/wso2/siddhi/extension/regex/LookingAtFunctionExtensionTestCase.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
new files please add the current year.. Change it in all the new palces

Reply:
fixed in 94099d3